solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -215,18 +215,21 @@ export default {
             this.fetch_content(this.title, this.part, location)
         },
         parseHash(locationHash) {
+            if (window.location.hash === "#main-content") return "";
```

Comment:
This fixes bug where "Skip To Main Content" would confuse the hashchange listener and cause issues.  Now, skipping to main content is simply ignored.